site/en/gemini-api/docs/vision.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<blockquote>Images greater than 20MB cannot be handled</blockquote><p> </p><p>This isn't strictly true, is it? AIUI, it's requests of 20MB or more, so 5x4MB images would hit this limit.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/453/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
You're right, I'd fixed it later in the doc but not there. Fixed now.